### PR TITLE
[DONE]: Translating "Quick Start / Index Page"

### PR DESCRIPTION
**Pull request by Yarob50**
🚧 Work in progress

### CODE REVIEW

**Comment by Yarob50** [comment]
Hi guys, I need your help..
as you can see in the deployed version the direction is not set to `rtl` so not only does it align the text to the left side, but also the order of the sentence is missed.

So I was wondering about the right place to fix the direction. should that be done by my side or what?
image

anyone?

**Comment by Yarob50** [comment]
So as mentioned in the last commit, the initial translation of all the sections of the page is complete, I kept the English version of the content, for now, to make it easier for the reviewer to know what was the original sentence and compare it with the translation I made, so I will remove it later at the end.
Also, it still looks messed up with the current `LTR` direction of the website which I am waiting for the response on.
Waiting for your notes to wrap it up.

**Comment by Yarob50** [comment]
@A7med3bdulBaset 
I guess this one should be added as part of the codebase to be there for anyone contributing. Thanks Ahamed.

**Comment by A7med3bdulBaset** [comment]
Yesterday, I made a PR for that #470 with some improvements

**Comment by Yarob50** [comment]
@3imed-jaberi, the code is now ready for review.
thanks.

**Review by A7med3bdulBaset — APPROVED** [review]

File: package-lock.json
[the remark this answers is not in the document]
Comment (Yarob50) [reply package-lock.json:1]: I see

File: src/content/learn/index.md [hunk 1]
```diff
@@ -296,29 +319,33 @@ function MyButton() {
...
 ```
 
-Notice how `onClick={handleClick}` has no parentheses at the end! Do not _call_ the event handler function: you only need to *pass it down*. React will call your event handler when the user clicks the button.
+لاحظ أن `onClick={handleClick}` لا تحتوي على أقواس الاستدعاء في النهاية! لا تقم باستدعاء معالج الحدث، يكفيك أن تقوم بتمرير اسمها فقط. حيث أن React ستقوم باستدعاء دالة معالجة الحدث عندما يقوم المستخدم بالضغط على الزر.
+
+
+## تحديث الشاشة {/*updating-the-screen*/}
 
-## Updating the screen {/*updating-the-screen*/}
+غالبا ستكون بحاجة أن يقوم المكوّن "بتذكر" المعلومات و عرضها على الشاشة. فمثلا، قد تحتاج إلى عرض عدد المرات التي تم فيها الضغط على زر ما. لتنفيذ ذلك قم بإضافة ما يعرف ب *حالة* أو الـ *state* الخاصة بالمكون:
```
[the remark this answers is not in the document]
Comment (Yarob50) [reply src/content/learn/index.md]: أهنيك والله على تمكنك من اللغة العربية، شكرا لك

**Pull request merged**